tools/windows-node-installer/go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/openshift/windows-machine-config-bootstrapper/tools/windows-no
 go 1.13
 
 replace (
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20200205145930-e9d93e317dd1 // OpenShift 4.3
-	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a // OpenShift 4.3
-	k8s.io/api => k8s.io/api v0.16.7
-	k8s.io/apimachinery => k8s.io/apimachinery v0.16.7
-	k8s.io/client-go => k8s.io/client-go v0.16.7
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200422081840-fdd1b0c14c88
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200422192633-6f6c07fc2a70
```

Comment:
Please add a comment saying which OpenShift version this corresponds to.

Reply:
added